app/code/Magento/Sales/Cron/CleanExpiredQuotes.php

```diff
@@ -17,28 +20,37 @@ class CleanExpiredQuotes
     /**
      * @var StoresConfig
      */
-    protected $storesConfig;
+    private $storesConfig;
```

Comment:
Please, revert to protected, we are not allowed to change vars visibility according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)